store/src/test/java/com/jinyframework/keva/store/StoreTest.java

```diff
@@ -0,0 +1,113 @@
+package com.jinyframework.keva.store;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.Serializable;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import lombok.*;
+
+public class StoreTest {
+    private static final String storeName = "junit-test";
+    private static NoHeapStoreManager manager = null;
+    private static NoHeapStore noHeapStore = null;
+
+    @BeforeAll
+    static void initStore() {
+        try {
+            manager = new NoHeapStoreManager();
+            manager.createStore(storeName, NoHeapStore.Storage.IN_MEMORY, 8);
+            noHeapStore = manager.getStore(storeName);
+        } catch (Exception e) {
+            fail(e);
+        }
+    }
+
+    @Test
+    void getSetString() {
+        String testName = "getSetString";
+        val setAbc = noHeapStore.putString(testName + "abc", "123");
+        val getAbc = noHeapStore.getString(testName + "abc");
+        val getNull = noHeapStore.getString(testName + "notExisted");
+        assertTrue(setAbc);
```

Comment:
Please use assertEquals for string result comparison. It's makes log more meaningful